_pages/early-planning.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: Your ATO
+title: ATOs, Early Planning
 ---
 
-An Authority to Operate (ATO) is a complicated security review process that is required for deploying any web service on the public web. [Learn more about ATOs](../ato/).
+An Authority to Operate (ATO) is a complicated security review process that is required before you can deploy anything on the public web. [Learn more about ATOs](../ato/).
 
-First, make sure that your project is not more complicated than the typical Open Data ATO. The following are signs that your ATO process will be more complicated and you should consult Noah when you begin planning:
+First, make sure that your project is not more complicated than the typical open data ATO. The following are signs that your ATO process will be more complicated and you should consult with Noah Kunin when you begin planning:
```

Comment:
I was using "Open Data" as a proper noun here, referring to the ATO level it gets, so I think caps makes sense...?